src/components/Article.jsx

```diff
@@ -1 +1,27 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (props) => {
+  const {
+    title,
+    date,
+    text,
+  } = props;
+
+  return (
+    <>
+      <h2 className="title" data-cy="title">{title}</h2>
+
+      <p className="subtitle">
```

Comment:
you can keep date stuff variables as it was at start) but your version is ok)

Reply:
yeah, let's move it to a variable